.github/workflows/ios_ci.yml

```
@@ -24,7 +24,7 @@ jobs:
     steps:
       - name: 🔄 Checkout source code
         uses: actions/checkout@v4
-
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)